File: src/Around.Frontend/Around-Frontend/src/app/model/copter-aggregate.ts
export class CopterAggregate {
    constructor(
        public name: string,
        public status: number,
        public latitude: number,
        public longitude: number,
        public costPerMinute: number,
        public brandId: number,
        public maxSpeed: number,
        public maxFlightHeight: number,
        public control: number,
        public droneType: number) {
    }
}

File: src/Around.Core/Entities/CopterAggregate.cs
namespace Around.Core.Entities
{
    public class CopterAggregate
    {
        public string Name { get; set; }

        public int Status { get; set; }

        public double Latitude { get; set; }

        public double Longitude { get; set; }

        public double CostPerMinute { get; set; }

        public int BrandId { get; set; }

        public double MaxSpeed { get; set; }

        public double MaxFlightHeight { get; set; }

        public int Control { get; set; }

        public int DroneType { get; set; }
    }
}
